src/main/java/org/opentripplanner/routing/graph/Graph.java

```diff
@@ -1000,4 +1000,8 @@ public MultiModalStation getMultiModalStation(FeedScopedId id) {
     public Collection<SimpleTransfer> getTransfersByStop(StopLocation stop) {
         return transfersByStop.get(stop);
     }
+
```

Comment:
I see that this method expose a `public` field, I would prefer that the field is made private. This method is exposed in the RoutingService using lombok delegation, while the field is not. So, removing this method is not an option.

Move tha JAvaDoc from the field to the get method. You may consider exposing the entire map, but it is better to do it the way you have - as long as there is a way to set the map values.